numpy/_core/src/multiarray/lowlevel_strided_loops.c.src

```diff
@@ -25,20 +25,6 @@
 
 #include "umathmodule.h"
 
-/*
- * x86 platform works with unaligned access but the compiler is allowed to
- * assume all data is aligned to its size by the C standard. This means it can
- * vectorize instructions peeling only by the size of the type, if the data is
- * not aligned to this size one ends up with data not correctly aligned for SSE
- * instructions (16 byte).
- * So this flag can only be enabled if autovectorization is disabled.
- */
-#if NPY_ALIGNMENT_REQUIRED
-#  define NPY_USE_UNALIGNED_ACCESS 0
```

Comment:
`NPY_USE_UNALIGNED_ACCESS` is used elsewhere in this file.

Reply:
good point, I'll bring it back

Reply:
Since it is always zero, I think you can remove it and update the code.